.clangd

```diff
@@ -8,3 +8,7 @@ CompileFlags:
     - -std=c11
     - -Ideps/sokol
     - -Ideps/cimgui
+    - -Invapi
+    - -D_COSMO_SOURCE
+    - -I/opt/cosmos/include
+    - -include libc/integral/normalize.inc
```

Comment:
Is this relative to the project?

Reply:
without this <cosmo.h> and friends won't work. (missing a few internal macros and typedefs)